SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -549,6 +549,56 @@ FunctionDef {
 		^().putPairs(this.keyValuePairsFromArgs)
 	}
 
+
+	prMakePerformableEnvir {|argsArray=([]), keywordArgsEvent=(()), dropFirstN=0|
+		var argNamesNoThis = argNames.drop(dropFirstN);
+		var setOfAddedNames = Set();
+		var out = ();
```

Comment:
better `envir = ()` (this helps reading because the method is called `prMakePerformableEnvir`).